src/pool-bin/BinPoolManager.sol

```diff
@@ -303,7 +303,7 @@ contract BinPoolManager is IBinPoolManager, ProtocolFees, Extsload {
 
     /// @inheritdoc IBinPoolManager
     function setMinBinSharesForDonate(uint256 minBinShare) external override onlyOwner {
-        if (minBinShare < 1e18) revert MinShareTooSmall();
+        if (minBinShare < 1e3) revert MinShareTooSmall();
```

Comment:
One suggestion , we can remove the minBinShare limit check in BinPoolManager, then add limit in PCS poolManagerOwner contract.

Reply:
yeh good call, i think we can 

1/ leave the updated check in PoolManager in this PR merged
2/ when the upcoming PR of `poolManagerOwner` contract come, then we can remove this check